.circleci/orb/orb.yml

```diff
@@ -74,9 +74,11 @@ commands:
     steps:
     - attach_workspace:
         at: /tmp/workspace
-    - run: curl -Ls https://github.com/nowsecure/auto-circleci-plugin/archive/1.2.0.tar.gz | tar -xzf - -C .
+    - run: curl -Ls https://github.com/nowsecure/auto-circleci-plugin/archive/1.2.1.tar.gz | tar -xzf - -C .
     - run:
-        command: cd auto-circleci-plugin-1.2.0 && gradle run -Dauto.url=<< parameters.auto_url >> -Dauto.token=<< parameters.auto_token >> -Dauto.dir=<< parameters.auto_dir >> -Dauto.file=<< parameters.auto_file >> -Dauto.group=<< parameters.auto_group >> -Dauto.wait=<< parameters.auto_wait >> -Dauto.score=<< parameters.auto_score >> -Dauto.username=<<parameters.auto_username>> -Dauto.password=<<parameters.auto_password>> -Dauto.show.status.messages=<<parameters.auto_show_status_messages>> -Dauto.stop.tests.on.status=<<parameters.auto_stop_tests_on_status>>
+        command: cp << parameters.auto_file >> auto-circleci-plugin-1.2.1/<< parameters.auto_file >>
+    - run:
+        command: cd auto-circleci-plugin-1.2.1 && ./gradlew run -Dauto.url=<< parameters.auto_url >> -Dauto.token=<< parameters.auto_token >> -Dauto.dir=<< parameters.auto_dir >> -Dauto.file=<< parameters.auto_file >> -Dauto.group=<< parameters.auto_group >> -Dauto.wait=<< parameters.auto_wait >> -Dauto.score=<< parameters.auto_score >> -Dauto.username=<<parameters.auto_username>> -Dauto.password=<<parameters.auto_password>> -Dauto.show.status.messages=<<parameters.auto_show_status_messages>> -Dauto.stop.tests.on.status=<<parameters.auto_stop_tests_on_status>>
```

Comment:
I don't see any issue replacing `gradle` with `./gradlew` but did you see any issues using gradle as it should be in the path of orb image.

Reply:
I found that it was not in the path of the orb image